src/Dashboard.jsx

```diff
@@ -1,51 +1,92 @@
-import React from 'react';
-import Widget from './components/Widget';
-import ListWidgetContainer from './containers/List/ListWidgetContainer';
+import React, { Component } from 'react';
 
 import './styles/App.css';
-import WeatherWidgetContainer from './containers/Weather/WeatherWidgetContainer';
+import axios from 'axios';
+import Tube from './containers/TubeStatus/Tube/Tube';
+import Tram from './containers/ManchesterTramStatus/Tram/Tram';
+import GalleryWidgetContainer from './containers/GalleryWidget/GalleryWidgetContainer';
+import Widget from './components/Widget';
 import Header from './containers/Header';
+import ListWidgetContainer from './containers/List/ListWidgetContainer';
 import Footer from './containers/Footer';
-import GalleryWidgetContainer from './containers/GalleryWidget/GalleryWidgetContainer';
-import Tube from './containers/TubeStatus/Tube/Tube';
 import OverheardWidgetContainer from './containers/Overheard';
-import RemindersWidgetContainer from './containers/Reminders';
 import WifiPasswordContainer from './containers/WifiPassword';
+import WeatherWidgetContainer from './containers/Weather/WeatherWidgetContainer';
+import RemindersWidgetContainer from './containers/Reminders';
+
+const BASE_API_URL = 'http://etdash-env.stujddpjmb.eu-west-2.elasticbeanstalk.com';
+
+async function determineLocationFromCoordinates(coordinates) {
+    if (coordinates.latitude && coordinates.longitude) {
+        const response = await axios.get('https://locationiq.org/v1/reverse.php', {
+            params: {
+                key: process.env.REACT_APP_LOCATIONIQ_TOKEN,
+                lat: coordinates.latitude,
+                lon: coordinates.longitude,
+                format: 'json'
+            }
+        });
+        return response.data.address.city;
+    }
+    return undefined;
+}
+
+class Dashboard extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { location: '' };
+    }
+
+    componentDidMount() {
+        // eslint-disable-next-line no-undef
+        navigator.geolocation.getCurrentPosition(position => {
+            determineLocationFromCoordinates(position.coords).then(city => {
+                this.setState({ location: city });
+            });
+        });
+    }
+
+    render() {
+        let locationBasedComponent;
+        switch (this.state.location) {
+            case 'London':
+                locationBasedComponent = <Tube />;
+                break;
+            case 'Manchester':
+                locationBasedComponent = <Tram href="http://localhost:8081/transport-manchester/status" />;
```

Comment:
Should this be localhost? I've not entirely read through I'm just checking.

Reply:
You're right should be the BASE_API forgot to update.